README.md

```diff
@@ -26,12 +26,16 @@
 </p>
 
 <p align="center">
-  <img alt="Twitter Follow" src="https://img.shields.io/twitter/follow/tramlinehq?style=social">
-  <img alt="Discord" src="https://img.shields.io/discord/974284993641725962?style=plastic">
+  <a href="https://twitter.com/tramlinehq/">
+    <img alt="Twitter Follow" src="https://img.shields.io/twitter/follow/tramlinehq?style=social">
+  </a>
+  <a href="https://discord.gg/u7VwyvBV2Z">
+    <img alt="Discord" src="https://img.shields.io/discord/974284993641725962?style=plastic">
+  </a>
```

Comment:
suggestion: should we open the links on a new tab so that people don't have to close the github page for external links? add `target="_blank" rel="noopener noreferrer"` to the tags.